app/src/App.js

```diff
@@ -152,13 +203,34 @@ const App = () => {
   // }
   // console.log(set);
 
+  const [graph, setGraph] = React.useState("Birthday Party");
+
+  const onSave = async (e) => {
+    let nodes = tasksx,
+      links = arrows;
+    let saveData = { graph, nodes, links };
+    e.preventDefault(); // prevents refreshing
+    let canAdd = saveData.nodes.length > 0 && graph.length > 0;
+    if (canAdd) {
+      await apiClient.addTask(saveData);
+      // loadTasks();
+      // setTask("");
+    }
+  };
+
   return (
     <>
       <header>
         <h1>Prerequisite Tree</h1>
       </header>
 
-      <main className="App">
+      <main
+        className="App"
+        // style={{
+        //   backgroundImage: `url(${image.hdurl})`,
+        //   backgroundSize: "cover",
```

Comment:
I see you're already using `backgroundSize: "cover"`, that's good. 
You can also add `backgroundPosition: "center"` if you want a little more control of the positioning.
:]